src/Neo.Compiler.CSharp/CompilationContext.cs

```
@@ -149,6 +149,12 @@ private void Compile()
 
         internal static CompilationContext Compile(IEnumerable<string> sourceFiles, IEnumerable<MetadataReference> references, Options options)
         {
+            if (IsSingleAbstractClass(sourceFiles))
+            {
+                Console.Error.WriteLine("The source file does not contain a valid neo Smart Contract");
+                Environment.Exit(-2);
```

We don't have a list of error codes, I think that we should throw the exception